preview-src/pullRequestOverviewRenderer.ts

```diff
@@ -591,11 +591,24 @@ class ReviewNode {
 						});
 					}
 
+					let outdated = comments[0].position !== comments[0].original_position;
```

Comment:
in other places we check if position is null to determine if a comment is outdated. I think we should do that here too

Reply:
done